### DISCUSSION TITLE
Welcome to ossph.org Discussions!

### DISCUSSION THREAD

**Discussion by jofftiquez** [OSSPhilippines/ossph.org#31] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nicklm0** [comment]
Hi @jofftiquez good morning My name is Nick Lemoncito, and I'm reaching out because I'm interested in joining OSSPH. I have been following the organization and its open-source projects, and I'm impressed by the impactful work being done by the community.

I have a strong passion for open-source software and believe in its potential to foster collaboration, innovation, and positive change. With my skills and experience in  I am eager to contribute to the projects at OSSPH.

**Reply by jofftiquez** [reply]
Hi @nickaldwin thank you! Please send me a DM in discord so we can talk about this <3

**Comment by nicklm0** [comment]
@jofftiquez thank you

**Comment by melio0504** [comment]
Two years later, here I am—hello! You can call me melio.

I’m excited to step into the world of open source, starting with small contributions and eventually working on bigger things.

I hope this community continues to grow and thrive in the years to come.

**Reply by jofftiquez** [reply]
Welcome @melio0504 <3 🎉 Thank you for your wishes!
